Improved test (#5)
PhraseePhoundry/front-end-test: README.md

Feel free to use any technology available in the React environment to achieve the results.

## Design

> The login page should look like, but do not make it pixel perfect.

Reviewer: ```suggestion
The login page should look like the following image, but doesn't have to be pixel perfect.
```

        "username": "test.user@phrasee.co",
        "password": "testpassword"
    }
   
3.  After a successful response has been received, use it to render a page listing hospital patients following the design attached in 'digital-hospital-global-patients.jpg'.
4.  This page should not be available to users who are 'unauthenticated'. Those users should be presented with the login page instead.

> 5.  Favor the use Redux to manage the some of the state of the app.

Reviewer 2: This is a bit weird grammatically, can we change it to "Whilst not a strict requirement, it is preferable that you use Redux to manage at least some of the state of the app." please?

Author: Thanks, done
